static/app/components/repositoryRow.tsx

```diff
@@ -105,31 +162,23 @@ class RepositoryRow extends Component<Props> {
                 )}
               </div>
             </RepositoryTitleAndUrl>
-
-            <Tooltip
-              title={t(
-                'You must be an organization owner, manager or admin to remove a repository.'
-              )}
-              disabled={hasAccess}
-            >
-              <Confirm
-                disabled={
-                  !hasAccess ||
-                  (!isActive && repository.status !== RepositoryStatus.DISABLED)
-                }
-                onConfirm={this.deleteRepo}
-                message={t(
-                  'Are you sure you want to remove this repository? All associated commit data will be removed in addition to the repository.'
-                )}
-              >
-                <Button
+            {repository.provider.id === 'integrations:custom_scm' ? (
```

Comment:
I thought about gating this by the flag as well but we don't already have the `organization` here and I think because the API is gated with both (the provider check and the flag check) I think this might be okay. Also, when we delete integrations we disabled the repositories (which should also hide the edit from the UI). Willing to be persuaded if someone thinks otherwise.

Reply:
@MeredithAnya its trivial to get the organization props with the `withOrganization` HOC. I think it might be a good idea to check for the flag.

```
export default withOrganization(RepositoryRow);
```

Reply:
@NisanthanNanthakumar yeah I've used that before just thought it wasn't super needed. However probs better in the long run to have that consistency so I've added it in.